app/templates/components/forms/login-form.hbs

```diff
@@ -49,15 +49,17 @@
               <div class="ui center aligned segment basic">
                 <a href="{{href-to 'reset-password'}}" class="text muted"> {{t 'Forgot your password ?'}}</a>
               </div>
-              {{#unless noSocial}}
-                <div class="ui horizontal divider">
-                  Or
-                </div>
-                <button type="button" class="ui facebook button" {{action 'auth' 'facebook'}}>
-                  <i class="facebook icon"></i>
-                  {{t 'Login with Facebook'}}
-                </button>
-              {{/unless}}
+              {{#if false}}
```

Comment:
What is this?

Reply:
@ritikamotwani I was getting template limiting error so I thought travis check might not pass. 
![Screenshot 2019-04-07 at 8 15 31 PM](https://user-images.githubusercontent.com/31013104/55685394-3e216d80-5973-11e9-9c12-c6763581716c.png)
But I have made the changes.

Reply:
@harsh-jindal This isn't what you should be doing. You must be doing it wrong. Please read this https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-html-comments.md. Try to find the solution to the errors before trying alternate solution.